packages/collector/test/tracing/cloud/aws-sdk/v2/sqs/receiveMessage.js

```diff
@@ -6,43 +6,74 @@
 'use strict';
 
 const instana = require('../../../../../../')();
+
 const express = require('express');
-const port = process.env.APP_PORT || 3216;
-const agentPort = process.env.INSTANA_AGENT_PORT || 42699;
 const request = require('request-promise');
 const { sendToParent } = require('../../../../../../../core/test/test_util');
-
-const app = express();
-
```

Comment:
Re-arranged imports and variable initialization a bit, these statements have not actually been deleted, just moved.